### ISSUE TITLE
Carat for endnotes too small a target

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4446]
split from 
- #4245 
> ... all [endnote] links being 'carat' ^ which is also too small a target for accessible selection.

Examples from #4194

- **Sample 2 (Material Theme) , item 49.** '] used with the carat as the only link text within notes are problematic. All of them end with these, so there is nothing to differentiate them, and repeated 'carat, link' is unhelpful when traversing the document by links.

- **Sample 3 (Clean Theme), item 19.** the end not link back to the original reading position is not contextual (just read as 'carat') such that all end notes have links that are read out identically.... imagine navigating by link 'carat, link, carat, link, carat, link...'

Not observed in OLH theme during this testing, but the template should be reviewed alongside the changes to Material and Clean to make sure.
